spec/requests/all_casa_admins_spec.rb

```diff
@@ -45,14 +45,14 @@
     before { sign_in admin }
 
     context "with valid parameters" do
-      let(:params) {
+      let(:params) do
```

Comment:
Don't think this is necessary either.

Reply:
I usually see `do` used over `{` on multi-line blocks and I thought that standardrb enforced it. Is our linter skipping our specs? And if so, is that intentional? They're not in the ignore file

Reply:
Not sure about that (re: standard). I'm not sure how much it digs into rspec yet. I don't really care, I'm just not in favor of making a change like this just because. But 🤷‍♀️ this is why I didn't mark them as blocking.

Reply:
`let` and other rspec-isms are ignored by Standard's semantic blocks https://github.com/testdouble/standard/blob/master/config/base.yml#L745